tests/phpunit/tests/widgets.php

```diff
@@ -686,6 +686,61 @@ public function test_wp_widget_get_settings() {
 		$this->assertArrayNotHasKey( 0, $never_used );
 	}
 
+	/**
+	 * @ticket 54677
+	 *
+	 * @covers WP_Widget::get_settings
```

Comment:
As per `@covers` [annotation document](https://phpunit.readthedocs.io/en/9.5/annotations.html#covers) the recommended method is `@covers ::functionName (recommended). We should follow it and other tests also follow this in the current file.
```suggestion
	 * @covers ::get_settings
```

Reply:
@mukeshpanchal27 That's just not true. The change you are suggesting is incorrect.

Using `@covers ::get_settings` would get PHPUnit chasing after a _global_ function called `get_settings()`, which doesn't exist.

The `@covers ::get_settings` annotation for class methods only works when combined with a class-level `@coversDefaultClass WP_Widget` annotation, which this test class doesn't have.

Reply:
Make sense. Thanks for clarification @jrfnl